src/libraries/System.Linq.Queryable/src/System/Linq/EnumerableRewriter.cs

```diff
@@ -136,21 +136,11 @@ private static Type GetPublicType(Type t)
                 return typeof(IEnumerable);
             return t;
 
-            [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2075:UnrecognizedReflectionPattern",
-                Justification = "The IGrouping<,> is kept since it's directly referenced here" +
-                    "and so it will also be preserved in all places where it's implemented." +
-                    "The GetInterfaces may return less after trimming but it will include" +
-                    "the IGrouping<,> if it was there before trimming, which is enough for this" +
-                    "method to work.")]
+            // IL2075
```

Comment:
What's the value/point of these comments in this file?

Reply:
On second thought they indeed bring little value, removed them.